src/mainField/ButtonFeed.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+const ButtonFeed = ({ setFoodPercent, setAge, age, setButtonFeedEntered }) => {
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    setButtonFeedEntered(true);
+    setFoodPercent(100);
```

Comment:
better not use "magic numbers", instead consider using constants like `MAX_FOOD_PERCENT`

Reply:
use initialFoodPercent instead "magic numbers"